### ISSUE TITLE
Same attribute is not shown

### ISSUE THREAD

**Issue by javmonisu** [jmurty/java-xmlbuilder#11]
Hello,
I have been using your library a while and I am dealing with an issue. Somehow, don't know why, some attributes of some elements are not shown. I have written the same attribute in many elements and it is shown in the first , but not in the others.
Thanks

**Comment by javmonisu** [comment]
The string is something like
e("xmlns:xxxx","xxx:xxx:xxxxxx:xx:xxx:1.0:xxxxxxxx:xxxxx:xxxx#")

**Comment by javmonisu** [comment]
I have been tracing the issue and it seems it is a bug in the XmlBuilder .asString() /  .asString(Properties)  methods

**Comment by jmurty** [comment]
Is this a bug, or a deliberate and valid simplification of the rendered XML? java-xmlbuilder doesn't do anything special when rendering attributes, it lets the standard XML serialisation mechanism do the work.

The attributes you are setting are XML namespace definitions, which makes me think the serialisation step is probably stripping redundant namespace re-definitions in child elements that already have the definition in a parent. You could confirm whether special handling of XML namespacing explains this by changing the attribute names to something else ("test-xmlns:xxxx") to see if the behaviour changes.

If that isn't the case, you will need to provide some more details about how you are building the document, the output you are getting, and the output you expect.

**Comment by javmonisu** [comment]
I have realized it is not a bug in your library ( which is fantastic ;) ). I implemented a XML builder using the org.w3c.dom. libraries and the same behaviour happens again. 

I had to build XML in a structure where the namespaces are defined in a lot of elements, It didn't make any sense to me ( because if you define in the root element it will work fine)  , I agree with your reasoning about what cause this issue.

Thank you, not a bug :+1:
